app/models/waste_exemptions_engine/registration.rb

```diff
@@ -58,5 +60,11 @@ def renewable?
     def in_renewable_state?
       active? || expired?
     end
+
+    def valid_mobile_phone_number?
+      record = clone
```

Comment:
What is this cloning?

Reply:
I'm reusing active record validator here which populates the instance error messages list with an error. My worry is that if I do that on an actual instance, this might cause issues while updating user details (because the error list is not empty).  This method does not suppose to perform validation, but just to check if phone number is valid mobile phone.